src/typing/flow_js.ml

```diff
@@ -5555,10 +5555,10 @@ let rec __flow cx ((l: Type.t), (u: Type.use_t)) trace =
       add_output cx ~trace FlowError.(EInternal (loc, ShadowWriteComputed))
 
     (* LookupT is a non-strict lookup *)
-    | (ObjProtoT _ |
-       FunProtoT _ |
-       DefT (_, MixedT (Mixed_truthy | Mixed_non_maybe))),
-      LookupT (_, NonstrictReturning t_opt, [], _, _) ->
+    | (ObjProtoT reason |
+       FunProtoT reason |
+       DefT (reason, MixedT (Mixed_truthy | Mixed_non_maybe))),
+      LookupT (reason_op, NonstrictReturning t_opt, [], _, _) ->
```

Comment:
I can do that, but I would argue that the pattern below https://github.com/facebook/flow/pull/4547/files/a11ccd0d4b85a1f59ea163a6203bd42c03e4f693#diff-f14393048a5695c25d5b2c97f6eca481R5576 of `RProperty Some name, RObjectType` it properly. Basically saying if `obj.prop` is non strict, trigger the error. The exercise above about making this a global error bore this out.